src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Resources/xlf/Strings.it.xlf

```diff
@@ -154,7 +154,7 @@
       </trans-unit>
       <trans-unit id="InvalidAssemblyVersion">
         <source>MC1005: Invalid AssemblyVersion detected: {0}.</source>
-        <target state="new">MC1005: Invalid AssemblyVersion detected: {0}.</target>
+        <target state="translated">MC1005: la versione dell'assembly rilevata {0} non è valida.</target>
```

Comment:
Most of the other translations do not alter `AssemblyVersion`.  Those would be correct to me.  `AssemblyVersion` is a well-known MSBuild property and would not be used differently in other languages as far as I know.  So it should not be translated here.

Reply:
PR updated with fixed version of string.